src/NuGet.Clients/NuGet.CommandLine/NuGetResources.resx

```diff
@@ -3047,474 +2990,6 @@ To prevent NuGet from downloading packages during build, open the Visual Studio
   <data name="InstallCommandPackageAlreadyExists_cht" xml:space="preserve">
     <value>已安裝封裝 "{0}"。</value>
   </data>
-  <data name="SourcesCommandNoSources_csy" xml:space="preserve">
```

Comment:
Need to think through the localization impact of this string table change.
Moved the english strings into nuget.commands.dll -- and was hoping standard localization would work. But NuGet.exe may be a different beast for loc.